### PR TITLE
[IMP] mail: change topbar thread description

### PR DESCRIPTION
**Pull request by zel-odoo**

### CODE REVIEW

File: addons/mail/static/src/new/discuss/discuss.xml [hunk 1]
```diff
@@ -19,9 +19,12 @@
...
                         <div class="flex-shrink-0 mx-2 py-2 border-start"/>
-                        <div class="px-1 border text-truncate border-light" t-att-title="thread.description">
-                            <t t-esc="thread.description"/>
-                        </div>
+                        <AutogrowInput
+                            class="{ general: 'o-mail-discuss-thread-description border', blur: 'bg-transparent border-light', hover: 'bg-white', focus: 'bg-white border-dark' }"
+                            onValidate.bind="updateThreadDescription"
+                            title="thread.description"
+                            value="thread.description"
+                        />
```
Comment (alexkuhn) [inline addons/mail/static/src/new/discuss/discuss.xml]: If you update description to empty string, then it's not possible to edit description anymore :-(

**Review by alexkuhn — CHANGES_REQUESTED** [review]

**Comment by alexkuhn** [comment]
Also make sure to reuse same history as target `master-discuss-refactoring`.
When I tried to rebase, somehow there was a conflict in `web/package.json`

**Comment by seb-odoo** [comment]
Do we still need this PR?

**Comment by alexkuhn** [comment]
Let's just drop the PR. If we need it at some point, let's just redo it from scratch

**Pull request closed without merging**